src/Angular.js

```diff
@@ -1602,6 +1622,11 @@ function angularInit(element, bootstrap) {
     }
   });
   if (appElement) {
+    if (!isAutoBootstrapAllowed) {
```

Comment:
This check could be moved at the top (unless we expect anyone to load angular via `resource:`/`chrome-extension:` in a non-extention page and bootstrap it manually.